src/features/peers/components/Peers.tsx

```diff
@@ -1,22 +1,27 @@
-import React from "react";
+import React, { FC, MouseEventHandler } from "react";
 import { Loading } from "../../loading";
 import { ErrorMessage } from "../../error-message";
 import { useDispatch } from "react-redux";
 import { updateStockAction } from "../../../actions";
 import "./Peers.css";
+import { PeersData } from "../redux/actions";
 
-export const Peers = ({ peers }) => {
+type PeersProps = {
+  peers: PeersData[] | null;
+};
+
+export const Peers: FC<PeersProps> = ({ peers }) => {
```

Comment:
Q: What is a FC and what characteristics does it take for this to be one?

Reply:
FC is a functional component as I will pass the prop inside.